### DISCUSSION TITLE
Proxy Cache dependency on upstream registry?

### DISCUSSION THREAD

**Discussion by strowi** [goharbor/harbor#18948] [Q&A]
Hi,

just came across this thought - does the proxy-cache in harbor work if the upstream registry is not available (e.g. still serve 
already cached images; like Gitlabs dependency-proxy)? 
Not sure how the failed HEAD-request is being handled by harbor?

Or would i need to create a full mirror?

regards,
strowi

**Comment by chlins** [comment]
@stonezdj Please help to confirm this, thanks.
